test/node-and-browser.js

```diff
@@ -290,4 +290,8 @@ describe('node and browser', function () {
     });
   });
 
+  it('should check that the database still exists before calling replicate', function () {
+    return expect(cluster._replicateDB("aaa", "db1"));
```

Comment:
I don't think the `expect()` call is doing anything here.

After refactoring per above, you can then create a spy and test:
```js
var createdAndReplicatedDB = false;
cluster._createAndReplicateDB = function () {
  createdAndReplicatedDB = true;
  return;
};

return cluster._replicateDB('aaa', 'db1').then(function () {
  createdAndReplicatedDB.should.eql(true)
});
```

Reply:
Thanks for sticking with it!